src/c++/perf_analyzer/genai-perf/genai_perf/wrapper.py

```diff
@@ -90,6 +91,8 @@ def build_cmd(args: Namespace, extra_args: list[str] | None = None) -> list[str]
         for arg, value in vars(args).items():
             if arg in skip_args:
                 pass
+            elif value is None:
```

Comment:
Why was this elif added?

Reply:
Because I removed _prune_args, so some namespace values could be None.

Reply:
i worry that we create the opportunity for other bugs with the removal of _prune_args.

Reply:
I think that is a valid concern. IMO I don't think there are much code that depends on this. The most critical one that I can think of is that it automatically removes args with None value so that when we are building PA command, we don't need to manually do it. But this is an easy fix. Also anything that depends on removed args needs to use `hasattr` to check if it's even in the namespace or not (otherwise it would throw error since the option does not exist in the namespace), but there's no more code that uses `hasattr` in our code.

Reply:
I am fine with this for now. I wanted to make sure we talked thru it.